src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -495,7 +498,20 @@ void CompatMidiRender::updateGateTime(const Instrument* instrument, int& gateTim
 
 void CompatMidiRender::createGraceNotesPlayEvents(const Score* score, const Fraction& tick, Chord* chord, int& ontime, int& trailtime)
 {
-    std::vector<Chord*> gnb = chord->graceNotesBefore();
+    auto hasNoPrebendGrace = [] (Chord* chord) -> bool {
+        return std::any_of(chord->notes().begin(), chord->notes().end(), [](Note* n) {
+            return !n->isPreBendStart();
+        });
+    };
+
+    std::vector<Chord*> gnbNotExcludingPrebends = chord->graceNotesBefore();
```

Comment:
You could call it with false to filter out the pre-bend note

Reply:
yeah, with true :) there are a bunch of similar flags in other places, and they are all inverted :)